sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/Customized/Models/UpdateModelSettingsOptions.cs

```diff
@@ -1,16 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-using Azure.Core;
-
-namespace Azure.Iot.TimeSeriesInsights
-{
-    /// <summary>
-    /// Options to update model settings. One of &quot;name&quot; or &quot;defaultTypeId&quot; must be set.
-    /// </summary>
-    [CodeGenModel("UpdateModelSettingsRequest")]
```

Comment:
This was exposed before since we were using options in API but now it is only taking string